### ISSUE TITLE
Refactor config module to be an lazy-load object

### ISSUE THREAD

**Issue by berinhard** [berinhard/pyp5js#94]
@isabarros raised a very valid point during [our discussion](https://github.com/berinhard/pyp5js/pull/93#discussion_r332812436) in the PR #93. To expand pyp5js configuration and also to be able to unit test the base `SKETCHBOOK_DIR` creation, we should change the existing `pytest.config` module to export a lazy-load object instead of giving direct access to its configuration constants. Django's [settings](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#use-of-django-conf-settings) should be the main guide as an approach to address that. The ideal import to use pyp5js configs would be:

```python
from pyp5js.config import conf
print(conf.SKETCHBOOK_DIR)
```

I think it's only worthwhile to work on this issue after #93 is merged into develop. This is because this new `conf` object implicates in a few code changes that would be better to do with a better test coverage for the web application.
